cmd/completion.go

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
We should use our own license headers. To fix this:
- Remove the Apache header
- Install `addLicense` by running `go get -u github.com/google/addlicense`
- Ensure the License header on every file: `addlicense -c "SIGHUP s.r.l" -v -l bsd cmd/`

Reply:
Whoops! Thanks for the review @angelbarrera92 ! Just pushed a commit fixing this. :)